src/daft-core/src/array/ops/list.rs

```diff
@@ -114,6 +116,43 @@ impl ListArray {
             )))
         }
     }
+
+    pub fn contains(&self, elements: &Series) -> DaftResult<BooleanArray> {
```

Comment:
Having quite a bit of trouble figuring out how to deduplicate code with the `contains` implementation for `FixedSizeList`